packages/arb-token-bridge-ui/src/components/TransactionHistory/helpers.ts

```diff
@@ -570,5 +570,5 @@ export function getTxHumanReadableRemainingTime(tx: MergedTransaction) {
   if (minutesLeft > 0) {
     return formattedMinutesLeft
   }
-  return 'Almost there...'
+  return 'less than a minute'
```

Comment:
We need to unify this because it's reused across different components and can be also shown as:
`Time left: less than a minute` or `less than a minute remaning`

Otherwise we end up with:
`Time left: Almost there...` or `Almost there... remaining`

Which sound wrong